### ISSUE TITLE
Inconsistent path treatment for `typeshedPath` and `typingsPath`. 

### ISSUE THREAD

**Issue by cshesse** [microsoft/pyright#355]
**Describe the bug**

There seems to be inconsistent and undocumented path treatment for `typeshedPath` and `typingsPath`.   

**To Reproduce**

Create a folder with the following `pyrightconfig.json`:

```
{
    "include": [
        "."
    ],
    "typeshedPath": "./typeshed",
    "typingsPath ": "./typings"
}
```
And two folders: `typeshed` and `typings`

Run `pyright --project .` from that folder:

```
pyright --project .
Loading configuration file at C:\p\misc\pyright_test7\pyrightconfig.json
typeshedPath ./typeshed is not a valid directory.
Searching for source files
No source files found.
Completed in 1.054sec
```

**Expected behavior**

I would expect that either both directories are found or neither.  It looks like `typeshedPath` is relative to some other location that is not the location of the config file.  If it was relative to the config file, this would make it easy for me to have it be correct no matter where the source code is located.

**VS Code extension or command-line**

```
pyright --version
pyright 1.1.4
```

**Comment by cshesse** [comment]
It looks like I copied and pasted "typingsPath" from the config doc so it actually has a space after the end.  I suspect this does not invalidate the issue though since it's just set to the default value.

**Comment by erictraut** [comment]
You're correct. The logic for normalizing and expanding the relative path for "typingsPath" was missing. This will be fixed in the next version of pyright.

The reason this wasn't discovered before is that typeshedPath normally points to a location outside of your workspace. Relative paths are therefore not typically used for this setting.

**Comment by cshesse** [comment]
Thanks for fixing it!

**Comment by erictraut** [comment]
This is now fixed in 1.1.5, which I just published.
